README.md

```diff
@@ -1,79 +1,88 @@
 
-![version](https://img.shields.io/badge/version-1-informational)
+Futher information on Specs : https://int-nit.github.io/AnDOChecker/
 
 [![PyPI license](https://img.shields.io/pypi/l/ansicolortags.svg)](https://pypi.python.org/pypi/ansicolortags/)[![Generic badge](https://travis-ci.org/Slowblitz/BidsValidatorA.svg?branch=master)](https://shields.io/)[![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/)
 
-## Table of Contents
+## Installation
+### Dependencies
 
-- [AnDOChecker](#andochecker)
-  - [Table of Contents](#table-of-contents)
-  - [Use](#use)
-    - [Clone](#clone)
-    - [General usage](#general-usage)
-    - [Specific usage](#specific-usage)
-  - [Development](#development)
-    - [Testing usage](#testing-usage)
-  - [Details](#details)
+AnDOChecker requires:
 
-## Use
+- Python (>= 3.6)
+- Pip
 
-### Clone
-
-- Clone this repo to your local machine using `https://github.com/INT-NIT/AnDOChecker.git`
+### User installation
+```bash
+> Pip3 install AnDOchecker
+```
 
-### General usage
+## General usage
 
 ```term
 usage: AnDOChecker.py [-h] [-v] path
 
 positional arguments:
   path           Path to your folder
 
-optional arguments:
-  -h, --help     show this help message and exit
+optional arguments:  -h, --help     show this help message and exit
   -v, --verbose  increase output verbosity
 
 ```
 
-### Specific usage
+## Specific usage
 
 ```bash
-$Python3 checker/AnDOChecker.py tests/ds001/data/Enya
+> Python3 checker/AnDOChecker.py tests/ds001/data/Enya
 ```
 
-<p align="center"><img src="doc/vids/AnDO_no_verbose.gif" /></p>
-
 OR verbose usage
 
 ```bash
-$Python3 checker/AnDOChecker.py -v tests/ds001/data/Enya
+> Python3 checker/AnDOChecker.py -v tests/ds001/data/Enya
 
 ```
 
-<p align="center"><img src="doc/vids/AnDO_with_verbose.gif" /></p>
+# Development
 
-## Development
+We welcome new contributors of all experience levels.  The
+[Development Guide][] has detailed information about contributing code,
+documentation, tests, and more. We've included some basic information in
+this README.
 
-### Testing usage
+Important links
+---------------
 
-```bash
-$cd checker/
-$python3.5 -m unittest discover -v
-```
+-   Official source code repo: https://github.com/INT-NIT/AnDOChecker
+-   Download releases: https://pypi.org/project/AnDOChecker/
+-   Issue tracker:https://github.com/INT-NIT/AnDOChecker/issues
+Source code
+-----------
+
+You can check the latest sources with the command:
+
+    git clone https://github.com/INT-NIT/AnDOChecker.git
+    
+Contributing
+------------
+
+To learn more about making a contribution to scikit-learn, please see
+our [Contributing guide][].
+
+Testing
+-------
+
+After installation, you can launch the test suite from outside the
+source directory (you will need to have `pytest` installed):
 
-## Details
+    pytest ando
 
```

Comment:
I don't think this is necessary... this is for developers and we're not here to explain developers how to perform unit tests

Reply:
Removed